changelog/unreleased/3837

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fix for database migration
```

Comment:
Users probably don't care about the technical issue, but the `Fix crash when upgrading from 2.18 ` or somehting like that, don't u think?